src/anomalib/metrics/f1_score.py

```diff
@@ -0,0 +1,18 @@
+"""F1 Score metric.
+
+This is added for convenience.
+"""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from torchmetrics.classification import BinaryF1Score as Metric
+
+
+class F1Score(Metric):
```

Comment:
In src/anomalib/metrics/auroc.py, you use `class AUROC(BinaryROC)`. I think that the same can be done here as well. Or the other way around as long as we have the consistent implementation